lib/matplotlib/ticker.py

```diff
@@ -1390,76 +1404,114 @@ def __init__(self, unit="", places=None, sep=" ", *, usetex=None,
         useMathText : bool, default: :rc:`axes.formatter.use_mathtext`
             To enable/disable the use mathtext for rendering the numbers in
             the formatter.
+        useOffset : bool or float, default: :rc:`axes.formatter.useoffset`
+            Whether to use offset notation. See `.set_useOffset`.
```

Comment:
This is an API change in that Engineering Formatters did not used to even support offsets, but now they default to using them.  I think the behavior here should be: 

 - accept {True, False, None}
 - default to False
 - None means "look at rcparams"